README.rst

```diff
@@ -79,8 +79,9 @@ Thanks for using Matrix!
 
 Synapse Installation
 ====================
-
-For details on how to install synapse, see `<INSTALL.md>`_.
+.. _federation:
```

Comment:
this doesn't seem to have worked. it may need more whitespace around it or something?